### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abhi99555**
Updated readme.md file with some proper changes.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2228"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Username: fossy
 Password: fossy
 ```
 
-**Note:** The test instance is not up to date with the latest release. The instance will reset every night at 2 am UTC and all the user uploaded data will be lost.
+**Note:** The test instance is not up to date with the latest release. The instance will reset every night at 2 am UTC and all the users' uploaded data will be lost.
```
Comment (NicolasToussaint) [inline README.md:93]: not quite convinced by the change, I think "user uploaded data" was OK as "user uploaded" kind of acts like an adjective for "uploaded data".
However, I would make it even simpler by changing to "all uploaded data will be lost." :-)

Comment (abhi99555) [reply README.md:93]: yaa,it will be fine

**Comment by shaheemazmalmmd** [comment]
@abhi99555 can you please make the changes as requested by @NicolasToussaint

**Comment by GMishx** [comment]
Closing the PR as there are no updates. @abhi99555 please reopen when needed.

**Pull request closed without merging**